base_layer/core/src/transactions/transaction/mod.rs

```diff
@@ -92,13 +100,15 @@ pub(super) fn hash_output(
     commitment: &Commitment,
     script: &TariScript,
     covenant: &Covenant,
+    version: TransactionOutputVersion,
 ) -> Vec<u8> {
     HashDigest::new()
         .chain(features.to_consensus_bytes())
         .chain(commitment.as_bytes())
         // .chain(range proof) // See docs as to why we exclude this
         .chain(script.as_bytes())
         .chain(covenant.to_consensus_bytes())
+        .chain((version as u8).to_le_bytes())
```

Comment:
Look at the comment in the genesis block, you need to update the MMR in the header.